.circleci/config.yml

```diff
@@ -37,11 +43,35 @@ jobs:
             - vendor
             - resources
             - app
+      - run:
+          name: Create implementation code by extending BaseIndex
+          command: |
+            cat <<'PHPSRC' > /var/www/mysite/www/app/src/MySolrIndex.php
+            <?php
+              use Firesphere\SolrSearch\Indexes\BaseIndex;
+              use SilverStripe\CMS\Model\SiteTree;
+              class MySolrIndex extends BaseIndex {
+              public function init() {
+                $this->addClass(SiteTree::class);
+                $this->addFulltextField('Title');
+                $this->addFulltextField('Content');
+              }
+              public function getIndexName() {
+                return 'MySolr';
+              }
+            }
+            PHPSRC
       - run: vendor/bin/sake dev/build
       #      No tests to run yet
       #      - run: vendor/bin/phpunit --coverage-clover=coverage.xml --log-junit ./coverage/junit.xml --coverage-html ./coverage/ -d memory_limit=512M tests/unit
 
       - run: vendor/bin/phpcs --standard=phpcs.xml.dist src
+      - run:
+          name: Run Solr Configure
+          command: |
+            mkdir -p /var/www/mysite/www/.solr && chmod -R 777 /var/www/mysite/www/.solr
```

Comment:
I think the folder needs to be owned by the `solr` user... Otherwise it can't write to it

Reply:
Ow, wait, no, Solr is not allowed to write outside it's container, that's the problem.

Reply:
You are correct I have to somehow map folders from primary container to solr or create a new docker image with solr on it bases on primary image

Reply:
I think the PostConfigStore would solve this for you?

Reply:
Thanks will look into that :)